README.md

```diff
@@ -5,6 +5,16 @@
 This repository is for the OpenMRS Dispensing App. For more information, please see the
 [Design Documentation](https://wiki.openmrs.org/display/projects/Dispensing+Design+Components).
 
+## Required privileges
+
+Note that following privileges need to be installed and assigned to roles:
+
+* "Task: dispensing.create.dispense" - Allows user to Dispense Medication
+* "Task: dispensing.create.dispense.andModifyDetails" - Allows user to modify the Quantity, Drug, Formulation and Dose Instructions (from the values specified in the Order / Medication Request) when Dispensing
+* "Task: dispensing.edit.dispense" - Allows user to edit an existing Medication Dispense
```

Comment:
Is there a world where we would want this to be different than the "create" privileges?  Would we ever want to prevent someone from editing something they created?  What if they made a typo?